shims/spark301/src/main/scala/com/nvidia/spark/rapids/shims/spark301/SparkBaseShims.scala

```diff
@@ -23,8 +23,8 @@ import com.nvidia.spark.rapids._
 import org.apache.arrow.memory.ReferenceManager
 import org.apache.arrow.vector.ValueVector
 import org.apache.hadoop.fs.Path
-
 import org.apache.spark.SparkEnv
```

Comment:
import groups look a bit off here, did you forget run `mvn verify`?

Reply:
yeah, put the newline back above to separate spark from others and newline below should be removed

Reply:
@abellina is right about my not running `mvn verify`. I'll restore the newline and check with `mvn verify`.

Reply:
This should be fixed now.